migrations/migrations/R__Create-Test-Data.sql

```diff
@@ -2024,6 +2050,30 @@ VALUES
   ) ON CONFLICT
 DO NOTHING;
 
+INSERT INTO
+  public.officer (
+    officer_guid,
+    user_id,
+    create_user_id,
+    create_utc_timestamp,
+    update_user_id,
+    update_utc_timestamp,
+    person_guid,
+    office_guid
+  )
+VALUES
+  (
+    '44123934-a2cf-4eae-88af-f682f7548f89'::uuid,
+    'MVESPRIN',
+    'FLYWAY',
+    '2024-01-22 22:16:16.754',
+    'FLYWAY',
+    '2024-01-22 22:20:48.186',
+    '141ebe0c-84c5-487d-8676-caee5de53b36'::uuid,
+    'c3d8519c-73cb-48a1-8058-358883d5ef4f'::uuid
+  ) ON CONFLICT
+DO NOTHING;
+
```

Comment:
You are going to want to include your auth_user_guid (01a5ad69-0675-4359-a0a7-909f55e2c67a) here otherwise the system won't pick you up as a target for assignment for complaints.